### PR TITLE
chore(docs): Remove warning

### PR DESCRIPTION
**Pull request by Nciklol**

### CODE REVIEW

**Comment by twisttaan** [comment]
You should also refactor https://github.com/Disploy/disploy/blob/560a2a99da17588f69362fe0ef87c62d0edae25a/apps/docs/docs/getting-started/cli-framework/prep-work/creating-a-new-project.md?plain=1#L9-L12 since `npx` is only supported now

**Review by twisttaan — CHANGES_REQUESTED** [review]
You should also refactor https://github.com/Disploy/disploy/blob/560a2a99da17588f69362fe0ef87c62d0edae25a/apps/docs/docs/getting-started/cli-framework/prep-work/creating-a-new-project.md?plain=1#L9-L12 since `npx` is only supported now

**Comment by Nciklol** [comment]
Not sure what to do to make the command fit better, should I merge the 2 parts of the page into one?

File: apps/docs/docs/getting-started/cli-framework/prep-work/creating-a-new-project.md [hunk 1]
```diff
@@ -4,18 +4,10 @@ sidebar_position: 2
 
 # Creating a new project
 
-:::warning
-
-`create-disploy-app` is not [yet implemented](https://github.com/Disploy/disploy/issues/50).
-
-:::
+## Start by running the command:
```
Comment (twisttaan) [inline apps/docs/docs/getting-started/cli-framework/prep-work/creating-a-new-project.md:7]: Maybe “Bootstrap a new project” with a description explaining how CDA gets you started with a basic template.

**Review by twisttaan — CHANGES_REQUESTED** [review]
Address reviews

**Review by twisttaan — APPROVED** [review]

**Comment by GodderE2D** [comment]
Why was yarn/npm create removed?

**Comment by twisttaan** [comment]
It's not supported.

**Comment by GodderE2D** [comment]
Is there any specific reason why?

**Comment by twisttaan** [comment]
Actually never mind, it does seem to work now. I tested it during the development of CDA, turns out yarn was just caching and an older version was published which was broken.
Screenshot 2022-12-14 at 11 32 31 pm

**Comment by twisttaan** [comment]
> Resolve ci issues (I have no idea how the ci works)

It's fine, the windows CLI tests always fail on pull requests because of some caching issue I need to investigate more. And the vercel preview failed because it's using a new configuration which isn't supported in the branch of this pull request because it needs to be updated w/ upstream.

**Review by GodderE2D — APPROVED** [review]

**Pull request merged**